storybook/storybook-docs/src/interactive-elements.stories.mdx

```diff
@@ -0,0 +1,66 @@
+
+### Relevante WCAG eisen:
+
+- [WCAG 2.4.4](https://www.w3.org/TR/WCAG22/#link-purpose-in-context): het is voor gebruikers duidelijk wat het betekent om op een link te klikken
+- [WCAG 2.4.6](https://www.w3.org/TR/WCAG22/#headings-and-labels): gebruik een label om interactieve elementen te beschrijven
+- [WCAG 2.5.3](https://www.w3.org/TR/WCAG22/#label-in-name): de toegankelijke naam van een element moet in ieder geval de zichtbare tekst van het element bevatten
+
+## Kleur
+
+Blauw is de interactieve kleur, deze wordt bijna altijd gebruikt voor interactieve elementen. Bij hover wordt dit donkerblauw. Een uitzondering hierop zijn error states, die gebruiken rood en donkerrood voor hover.
+
+Kleur mag niet de enige indicator van interactiviteit zijn, zodat mensen die moeite hebben met het waarnemen van verschillen in kleur toch kunnen zien dat een element interactief is.
+
+### Relevante WCAG eisen:
+
+- [WCAG 1.4.1](https://www.w3.org/TR/WCAG22/#use-of-color): kleur mag niet de enige manier zijn waarop je aangeeft dat iets interactief is
+- [WCAG 1.4.3](https://www.w3.org/TR/WCAG21/#contrast-minimum): tekst heeft voldoende contrast met de achtergrond
+
+## Formaat
+
+[Interactieve elementen moeten minstens 24 bij 24 pixels groot zijn.](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html)
+
+## Focus indicator
+
+Wij gebruiken grotendeels de standaard focus indicator van de webbrowsers. De enige aanpassing die we doen is de focus `outline` een `offset` geven van 2 pixels, zodat er ruimte zit tussen het element en de focus indicator. Hierdoor heeft de focus indicator ook voldoende contrast met donkere elementen.
+
+Let op bij het inrichten van een pagina dat de interactieve elementen in een logische volgorde staan, meestal van boven naar beneden en van links naar rechts. Hierdoor krijg je een logische focus / tabbing order.
+
+### Relevante WCAG eisen
+
+- [WCAG 2.4.3](https://www.w3.org/TR/WCAG22/#focus-order): interactieve elementen hebben een logische focus / tabbing order
+- [WCAG 2.4.7](https://www.w3.org/TR/WCAG22/#focus-visible): de focus indicator is zichtbaar
+- [WCAG 2.4.11](https://www.w3.org/TR/WCAG22/#focus-not-obscured-minimum): de focus indicator wordt niet volledig verborgen door content
+- [WCAG 2.4.12](https://www.w3.org/TR/WCAG22/#focus-not-obscured-enhanced): de focus indicator wordt niet deels verborgen door content
+- [WCAG 2.4.13](https://www.w3.org/TR/WCAG22/#focus-appearance): de focus indicator is minimaal 2 pixels breed en heeft voldoende contrast met het element dat focus heeft
+
+## Hover
+
+Gebruik verschillende, niet overlappende stijlen voor hover en focus. Als toetsenbordgebruiker is het belangrijk dat je weet welk element je activeert. Als de hover-variant (bijna) hetzelfde is als de focus-variant, dan weet je niet zeker welk element je zult activeren.
```

Comment:
Are keyboard users the only users without a pointing device? I think we can do more for our interactive elements using `any-pointer` media queries [@media/any-pointer](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/any-pointer)

Reply:
> Are keyboard users the only users without a pointing device?

Keyboard users usually also have a pointing device, that's what this section is about. A user can use both tab focus and hover, which is why they should look different.

> I think we can do more for our interactive elements using any-pointer media queries @media/any-pointer

I'm not a huge fan of `any-pointer`, it's a little brittle imo. I think we should solve any issues with innacurate pointing devices in the design itself, by making thing large enough by default (which we do, currently).